src/server/routes/index.js

```diff
@@ -45,11 +45,10 @@ router.get('/', async (req, res, next) => {
 	const {orm} = req.app.locals;
 	const numRevisionsOnHomepage = 9;
 
-	function render(entities) {
+	function render(results) {
 		const props = generateProps(req, res, {
 			homepage: true,
```

Comment:
If I remove the homepage, it changes the UI, It determines if there should be searchbox in navbar or not.